report/Introduction/ProjectProposal_eng.tex

```diff
@@ -0,0 +1,58 @@
+\section{Project Proposal}
+\subsection{Purpose}
+In agile processes there is a great focus on that requirements is not something that you just make, but something that you discover and improve throughout the project.
+To accommodate this method, there are different processes, with different ways accomplish it. 
+Where Scrum, works with a so called "Product Owner", which is a person representing the customer, either by being the actual customer, or someone with knowledge and authority (domain expert) to make decisions on behalf of the customer. 
+In Extreme Programming (XP) there is something similar, but called "Onsite Customer", which means that the customer must be present in the same room as the developers  and be available to answer questions and provide feedback on the quality of the product. 
+\\\\
+A common problem with both methods are that they require the customer leave their normal environment and be where the developers are. 
+This means that the customer or domain expert will have to recall the problem domain, and not be present in it.
+Since many of the details regarding the problem domain often comes as Tacit knowledge, which can make it hard to recall the details if the they are not problem domain. 
+
+The optimal scenario would be if the customer or domain expert could fill their role as Product Owner, while he or she were in the proper domain. 
+
+This would increase the quality of feedback.
+
+It is also a well known problem that customers are terrible at saying what they want, but it is very easy for them to say what they do not want.
+\\\\
+A phenomenon which Morten from this group have worked a lot with his 5 years of experience as a designer.
+In the world of design, to accommodate this problem you often present the customer with 3 proposals for designs, and ask the customer to point out what they like and do not like.
+This gives the designer some insight into what the customer actually wants and likes. 
+\\\\
+With these two problems concerning inclusion of the customer in the development process, it would be exiting to look at which other technologies could be used, that could allow the customer to express their needs while being in their problem domain while still being able to communicate with the developers. 
```

Comment:
```suggestion
With these two problems concerning inclusion of the customer in the development process, it would be exiting to look at which technologies could be used, that could allow the customer to express their needs while being in their problem domain while still being able to communicate with the developers. 
```